src/lib.rs

```diff
@@ -213,14 +209,11 @@ impl DocData {
         for def in defs.iter() {
```

Comment:
I think it might make more sense to turn this from `iter` to `into_iter` that way the clones below can be avoided.